src/screens/Swap/OperationDetails.js

```diff
@@ -62,8 +62,8 @@ const OperationDetails = ({ route }: Props) => {
   const swap = fromAccount.swapHistory.find(s => s.swapId === swapId);
   const status = Config.DEBUG_SWAP_STATUS || swap.status;
 
-  const fromCurrency = getAccountCurrency(fromAccount);
-  const toCurrency = getAccountCurrency(toAccount);
+  const fromCurrency = fromAccount && getAccountCurrency(fromAccount);
```

Comment:
Is the new condition because of flow? I see `fromCurrency.type` just below (line 71 - `const url = …`) that would crash if fromCurrency is undefined anyway I think?

Reply:
good catch i'll check the condition everywhere :+1: 
during testing i went in a case where fromAccount was null during a frame for a reason i dont know.
i prefer be sure than sorry here